modules/nw-route-specific-annotations.adoc

```diff
@@ -19,7 +19,7 @@ To create a whitelist with multiple source IPs or subnets, use a space-delimited
 [cols="3*", options="header"]
 |===
 |Variable | Description | Environment variable used as default
-|`haproxy.router.openshift.io/balance`| Sets the load-balancing algorithm. Available options are `random`, `source`, `roundrobin`, and `leastconn`. | `ROUTER_TCP_BALANCE_SCHEME` for passthrough routes. Otherwise, use `ROUTER_LOAD_BALANCE_ALGORITHM`.
+|`haproxy.router.openshift.io/balance`| Sets the load-balancing algorithm. Available options are `random`, `source`, `roundrobin`, and `leastconn`. The default value is `random`.| `ROUTER_TCP_BALANCE_SCHEME` for passthrough routes. Otherwise, use `ROUTER_LOAD_BALANCE_ALGORITHM`.
```

Comment:
If no annotation is added to the route, the route will have `leastconn` as the default which in turns comes via  `ROUTER_LOAD_BALANCE_ALGORITHM` proxy pod variable.  Hence the statement should be re-phrased to reflect the same.

Reply:
One correction is needed: `The default value is leastconn`.  The rest looks good.